subsys/net/ip/Kconfig

```diff
@@ -35,11 +35,26 @@ source "subsys/net/ip/Kconfig.ipv4"
 
 config NET_SHELL
 	bool "Enable network shell utilities"
-	select CONSOLE_SHELL
+	select SHELL
 	help
 	  Activate shell module that provides network commands like
 	  ping to the console.
 
+config NET_SHELL_DYN_CMD_COMPLETION
```

Comment:
Is this implemented / needed ?

Reply:
The reason for this setting is that if one wishes to autocomplete IPv6 addresses for example, then we need to allocate some space for these ip addresses. As each IPv6 address string can take up to 46 bytes of memory and we might have for example 8 of them defined, that would occupy over 300 bytes of memory. I made this configurable because of that so that if the device is limited with memory, then this dynamic cmd support can be turned off.